platform/android/export/export_plugin.cpp

```diff
@@ -2087,11 +2087,12 @@ Error EditorExportPlatformAndroid::run(const Ref<EditorExportPreset> &p_preset,
 
 	String tmp_export_path = EditorPaths::get_singleton()->get_cache_dir().path_join("tmpexport." + uitos(OS::get_singleton()->get_unix_time()) + ".apk");
 
-#define CLEANUP_AND_RETURN(m_err)                         \
-	{                                                     \
-		DirAccess::remove_file_or_error(tmp_export_path); \
-		return m_err;                                     \
-	}                                                     \
+#define CLEANUP_AND_RETURN(m_err)                                    \
+	{                                                                \
+		DirAccess::remove_file_or_error(tmp_export_path);            \
+		DirAccess::remove_file_or_error(tmp_export_path + ".idsig"); \
```

Comment:
It could throw an error if the file wasn't signed yet.

```suggestion
		if (FileAccess::exists(tmp_export_path + ".idsig") { \
			DirAccess::remove_file_or_error(tmp_export_path + ".idsig"); \
		} \
```

Reply:
But if export failed with no cresting APK at all? When removing file DirAccess will also throw an error. Should we do this check for both deletions then?